apps/mobile/src/screens/GroupWelcomePage/GroupWelcomePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { View, ScrollView } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import HyloHTML from 'components/HyloHTML'
+import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
+
+export default function GroupWelcomePage() {
+  const insets = useSafeAreaInsets()
+  const [{ currentGroup }] = useCurrentGroup()
+
+  return (
+    <ScrollView 
+      className='flex-1 bg-background'
+      contentContainerStyle={{
+        paddingTop: insets.top,
+        paddingBottom: insets.bottom,
+        paddingLeft: insets.left,
+        paddingRight: insets.right
+      }}
+    >
+      <View className='p-4'>
+        <HyloHTML 
```

Comment:
Did you get this going with <a> tags? It should work fine but you may need to look at other uses of HyloHTML and otherwise inspect the component itself to make sure you're giving what it needs to handle the links?

Reply:
sooo, doesn't look like any instances of HyloHTML do any different styling beyond one instance of passing in a marginBottom. The base component does seem to have a set of default styles passed in: `import { baseStyle, tagsStyles, classesStyles } from 'components/HyloHTML/HyloHTML.styles'` So maybe something is missing from there? I'm on the road between LA and SF right now, so just pulled over to get this branch up to date with yours

Reply:
I am not aware of what the issue is, you had just mentioned that <a>nchor tags were not working but wasn't sure what else you were seeing or experiencing. It sounds like something isn't doing what you expected here, but yes the styling is fairly fixed and not easy to override on a case-by-case basis. HTML in Mobile is a weird thing. So not sure you're flow, but either explore further and/or give me notes as to what is missing and/or quick instructions on how to test. Will add to todo backlog, whatever is there after release goes into tickets.